packages/owl-bot/src/configs-store.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {OwlBotLock, OwlBotYaml} from './config-files';
+
+export interface Configs {
+  // The body of .Owlbot.lock.yaml.
+  lock: OwlBotLock | undefined;
+  // The body of .Owlbot.yaml.
+  yaml: OwlBotYaml | undefined;
+  // The commit hash from which the config files were retrieved.
+  commitHash: string;
+  // The branch name from which the config files were retrieved.
+  branchName: string;
+  // The installation id for our github app and this repo.
+  installationId: number;
+}
+
+export interface ConfigsStore {
```

Comment:
I don't see another implementor of this interface - is there a reason to have it as an interface instead of just using the single exported class?  Is there a potential to have multiple config stores?  Trying to avoid Java-esque INTERFACE ADAPTER FACTORY PATTERN stuff.

Reply:
This interface was useful for splitting the work between Ben and I.